website/server/libs/payments.js

```diff
@@ -109,7 +117,19 @@ api.addSubToGroupUser = async function addSubToGroupUser (member, group) {
   if (member.isSubscribed()) {
     let memberPlan = member.purchased.plan;
     let customerHasCancelledGroupPlan = memberPlan.customerId === this.constants.GROUP_PLAN_CUSTOMER_ID && !member.hasNotCancelled();
-    if (customerIdsToIgnore.indexOf(memberPlan.customerId) !== -1 && !customerHasCancelledGroupPlan) return;
+    let ignorePaymentPlan = paymentMethodsToIgnore.indexOf(memberPlan.paymentMethod) !== -1;
+    let ignoreCustomerId = customerIdsToIgnore.indexOf(memberPlan.customerId) !== -1;
+
+    if (ignorePaymentPlan) {
+      txnEmail(FLAG_REPORT_EMAILS, 'invalid-payment-cancel', [
+        {name: 'USERNAME', content: member.auth.local.username},
+        {name: 'UUID', content: member._id},
+        {name: 'EMAIL', content: member.auth.local.email},
+        {name: 'PAYMENT_METHOD', content: memberPlan.paymentMethod},
```

Comment:
`purchased.plan` would be good to include too in case the user cancels the subscription and then waits long enough before emailing us for it to terminate and disappear from their account. We'll want to see the data for putting back their consecutive months, etc. The entire object can be converted to a single string if that's easiest - e.g. a string like this:

`{ "purchased" : { "plan" : { "planId" : "basic", "dateUpdated" : ISODate("2017-04-01T17:19:30.206Z"), "dateCreated" : ISODate("2014-09-12T19:50:24.320Z"), "customerId" : "someiosthing", "consecutive" : { "trinkets" : 42, "gemCapExtra" : 25, "offset" : 0, "count" : 30 }, "mysteryItems" : [ ], "gemsBought" : 50, "extraMonths" : 3, "quantity" : 1 } } }`